src/Raven.Client/Constants.cs

```diff
@@ -69,8 +68,34 @@ public class Windows
                 private Windows()
                 {
                 }
-
-                public static readonly int MaxPath = (int)typeof(Path).GetField("MaxLongPath", BindingFlags.Static | BindingFlags.NonPublic).GetValue(null);
+                
+                public static readonly int MaxPath = short.MaxValue;
+                
+                public static readonly string[] WindowsReservedFileNames = {
```

Comment:
Let's rename this to `ReservedFileNames` and make it `internal`. We are already in `Windows` anyway.

Reply:
done